cmd/helm/lint.go

```diff
@@ -93,10 +101,18 @@ func newLintCmd(out io.Writer) *cobra.Command {
 				}
 
 				for _, msg := range result.Messages {
-					fmt.Fprintf(&message, "%s\n", msg)
+					if client.Quiet {
```

Comment:
You can significantly reduce the cyclomatic complexity of this `if/else` block by rewriting it like this:

```
if !client.Quiet || msg.Severity > support.InfoSev {
    fmt.Fprintf(&message, "%s\n", msg)
}
```

truth table:

```
quiet | severity | output |
0     | 0         | 1     |
0     | 1         | 1     |
1     | 0         | 0     |
1     | 1         | 1     |
```